### DISCUSSION TITLE
request data type icons

### DISCUSSION THREAD

**Discussion by charleneyjh1** [twilio-labs/paste#3821] [Request: Icon] [answered]
### Description

Hello, we'll use some icons to represent different data types, such as string, number, boolean, array, object...
This will be used as a prefix in a dropdown selection, so that customers know what type their data is, and what type the destination fields accept. (example screenshot attached)
<img width="424" alt="Screenshot 2024-03-14 at 4 30 30 PM" src="https://github.com/twilio-labs/paste/assets/56697620/9508995e-2fce-43d4-867b-43f4f2a7526f">

We'd like to add the below icons to Paste:
String: [link](https://www.streamlinehq.com/icons/streamline-regular/interface-essential/text-formating/text-style)
Number: [Link](https://www.streamlinehq.com/icons/streamline-light/interface-essential/text-formating/text-format-numeric)
Boolean: [link](https://www.streamlinehq.com/icons/streamline-regular/interface-essential/controls/settings-toggle-horizontal
)
We weren't able to find good icons for array and object, and we are thinking of just using "[ ]" and "{ }" text in replacement of a real icon. What do you guys think?

Thank you for your help!

### Link to icon

https://www.streamlinehq.com/icons/streamline-regular/interface-essential/text-formating/text-style

### Icon type

- [X] UI icon
- [ ] Product icon

**Comment by nkrantz** [comment]
Hey @charleneyjh1 - we've seen your icon request but unfortunately it'll have to wait until Sarah is back from PTO as we have very limited capacity right now. Stay tuned for an update!

**Reply by nkrantz** [reply]
Btw do you have a timeline for when you'll need these data type icons?

**Reply by serifluous** [reply]
Hey @charleneyjh1 ! Can you submit a dependency request? https://airtable.com/appwvsQKyJHfnG5Sa/shrvMvL3BG6Zm0f9l

**Reply by charleneyjh1** [reply]
timeline is by End of April @nkrantz . Just submitted the request!

**Comment by nkrantz** [comment, marked as answer]
Hey @charleneyjh1 - the icons are now available!

**Reply by charleneyjh1** [reply]
yay thanks!
